styx-flyte-client/src/main/proto/README.md

```diff
@@ -0,0 +1,10 @@
+# Protos
+
+Styx uses these grpc and protobuf definitions to build the client and communicate with Flyte.
+
+The definitions are taken from the [flyteidl repo](https://github.com/lyft/flyteidl/tree/master/protos/flyteidl) and are built locally.
+
+## Update protos
+```
+cd into this directory
+sh update_protos.sh
```

Comment:
We should give this script `+x` and then just `./update_protos.sh`. The `sh` here is confusing.

Reply:
👍